Lib/test/test_code.py

```diff
@@ -256,10 +256,11 @@ def func():
             return x
         code = func.__code__
 
-        # different co_name, co_varnames, co_consts
+        # Different co_name, co_varnames, co_consts.
+        # Must have the same number of constants and
+        # variables or we get crashes.
```

Comment:
Is there an issue for these crashes?

Reply:
No. `replace` comes with a big red warning for this reason.
The code objects generated by this test would have segfaulted the VM if you tried to run them.